app/index.js

```diff
@@ -110,6 +110,19 @@ app.model = require('./model');
 app.controller = require('./controller');
 
 /**
+ * Log potentially unhandled promises
+ */
+// TODO: move over to testing suite
+if (app.config.env === 'development') {
+  process.on('unhandledRejection', function(reason, p) {
```

Comment:
i think what I meant was that binding to process.on() would catch ALL unhandled rejected promises, even ones that are not necessarily part of the webapp (for example, if one of our dependencies ignores rejected on promises on purpose, for example). But I guess since we're just logging, even if that happened it wouldn't affect anything (and it would be good to know anyway)

So I think you can safely disregard my comment :)